### PR TITLE
Display APIs on package list

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- display API & Events number on package list
- display number of packages on the Applications list
- ...

**Related issue(s)**

### CODE REVIEW

File: compass/src/components/Application/ApplicationDetails/ApplicationApiPackages/ApplicationApiPackages.test.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { render, queryByText } from '@testing-library/react';
+import ApplicationApiPackages from './ApplicationApiPackages';
+import { MockedProvider } from '@apollo/react-testing';
+import React from 'react';
+
+const mockApiPackages = [
+  {
+    name: 'package one',
+    description: 'It contains six spring tyne cultivators and a dwarf.',
```
Comment (dariadomagala) [inline compass/src/components/Application/ApplicationDetails/ApplicationApiPackages/ApplicationApiPackages.test.js]:
```suggestion
    description: 'It contains six spring tine cultivators and a dwarf.',
```
❗️

Comment (parostatkiem) [reply compass/src/components/Application/ApplicationDetails/ApplicationApiPackages/ApplicationApiPackages.test.js]: I suppose it's like `tire` and `tyre`.  Looking at Google, it seems to be written randomly.

Comment (parostatkiem) [reply compass/src/components/Application/ApplicationDetails/ApplicationApiPackages/ApplicationApiPackages.test.js]: image

Comment (dariadomagala) [reply compass/src/components/Application/ApplicationDetails/ApplicationApiPackages/ApplicationApiPackages.test.js]: 1 :
Screen Shot 2020-03-10 at 15 19 03
2:
Screen Shot 2020-03-10 at 15 19 29

Google prefers 'tine'

Comment (parostatkiem) [reply compass/src/components/Application/ApplicationDetails/ApplicationApiPackages/ApplicationApiPackages.test.js]: Dictionaries seem to share this view. Corrected.

**Pull request merged**